src/Uno.Extensions.Templates/content/unoapp-extensions/.template.config/template.json

```diff
@@ -266,22 +399,89 @@
       "datatype": "bool",
       "value": "tfm == 'net7.0'"
     },
-    "is-vsix": {
-      "type": "parameter",
+    "libarybasetargetframework": {
+      "type": "generated",
+      "generator": "switch",
+      "replaces": "$libarybasetargetframework$",
+      "parameters": {
+        "evaluator": "C++",
+        "datatype": "string",
+        "cases": [
+          {
+            "condition": "(tfm == 'net6.0')",
+            "value": "netstandard2.1"
+          },
+          {
+            "condition": "(tfm == 'net7.0')",
+            "value": "net7.0"
+          }
+        ]
+      }
+    },
+    "msftextlibversion": {
+        "type": "generated",
+        "generator": "switch",
+        "replaces": "$msftextlibversion$",
+        "parameters": {
+          "evaluator": "C++",
+          "datatype": "string",
+          "cases": [
+            {
+              "condition": "(tfm == 'net6.0')",
+              "value": "6.0.0"
+            },
+            {
+              "condition": "(tfm == 'net7.0')",
+              "value": "7.0.0"
+            }
+          ]
+        }
+      },
+    "blank-app-template": {
+      "type": "computed",
       "datatype": "bool",
-      "defaultValue": "false"
+      "value": "template-type == 'blank'"
+    },
+    "default-app-template": {
+      "type": "computed",
+      "datatype": "bool",
+      "value": "template-type == 'default'"
+    },
+    "use-default-nav": {
+      "type": "computed",
+      "datatype": "bool",
+      "value": "navigation == 'default'"
+    },
+    "use-frame-nav": {
+      "type": "computed",
+      "datatype": "bool",
+      "value": "navigation == 'frame'"
+    },
+    "not-frame-nav": {
+      "type": "computed",
+      "datatype": "bool",
+      "value": "navigation != 'frame'"
+    },
+    "HostIdentifier": {
+      "type": "bind",
+      "binding": "HostIdentifier"
     }
   },
   "primaryOutputs": [
     {
-      "path": "MyExtensionsApp.sln",
-      "condition": "!is-vsix"
+      "condition": "(HostIdentifier == \"dotnetcli\" || HostIdentifier == \"dotnetcli-preview\" || HostIdentifier == \"vs\")",
```

Comment:
When is this not true?

Reply:
this was some debugging code... thanks for catching it